multimodal/src/autogluon/multimodal/configs/pretrain/detection/default_runtime.py

```diff
@@ -0,0 +1,28 @@
+checkpoint_config = dict(interval=1)
```

Comment:
I have a quick question. If users just do `pip install autogluon` and wants to change the config, what should they do? Do they download the config from somewhere first, and then change hyperparameters manually? Thanks.

Reply:
We only retrieve the model structure and list of transforms from the config. Those parts are usually not changed and thus currently we do not support changing the config nor loading a customized config.